inference-engine/src/plugin_api/.clang-format

```diff
@@ -0,0 +1,27 @@
+BasedOnStyle: Google
```

Comment:
is it the same as for public headers? Is it possible to have a single file shared between multiple modules?